packages/bot-web-ui/src/utils/__tests__/download.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { LogTypes } from '@deriv/bot-skeleton';
+import { localize } from '@deriv/translations';
+import { getCurrentDateTimeLocale, getSuccessJournalMessage } from 'Utils/download';
+
+jest.mock('@deriv/translations', () => ({
+    localize: jest.fn((message, values) => {
+        let result = message;
+        if (values) {
+            Object.keys(values).forEach(key => {
+                result = result.replace(`{{${key}}}`, values[key]);
+            });
+        }
+        return result;
+    }),
+}));
```

Comment:
@mayuran-deriv do we need to mock `@deriv/translations`? There is a global mock for this in here `<root>/__mocks__/translation.mock.js`. Doesn't it work from there?

Reply:
thanks man this is removed.